src/components/Header/Header.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+import Logo from '../Logo/Logo';
+import './Header.css';
+
+/**
+ * The common Kids First navigation.
+ *
+ * Must pass an array of buttons for the navigation bar.
+ *
+ * The header should be included at the top of every Kids First application.
+ * It contains common functionality for all of Kids-First that should remain
+ * consistent, such as the user drop down and application navigation.
+ */
+const Header = ({ buttons, logoHref, onLogoClick, children }) => {
```

Comment:
thinking you can encapsulate the logo props into one props object `logoProps` this way as the component grows the props list won't get crazy long and it will be easy to tell what props get destructed where. 
```js
logoProps: {
 href: '#',
 onClick: () =>{}
}
```

@dankolbman how do you feel about making that a standard for molecular components?